File: cpp/semcomp-codeforces-2023/h/h.py
t = int(input())

for i in range(t):
    arr= [2, 1]
    tam = 3
    

    n = int(input())

    if n == 1:
        print(2)
        continue

    for j in range(n-2):
        o = j+3

        id=0
        while True:
            print(j)
            if arr[id] + arr[id+1] == o:
                tam += 2
                arr.insert(id+1, o)
                id+=1

            id +=1
            if id >= len(arr)-1:
                break
        
    print(tam)

File: cpp/semcomp-codeforces-2023/h/h.cpp
#include<bits/stdc++.h>

using std::cin;
using std::cout;

int main() {
    int t;
    cin >> t;
    

    for(int j = 0; j < t; j++) {

        std::vector<int> arr;
        long int tam = 3;

        arr.push_back(2);
        arr.push_back(1);
        long int n;

        cin >> n;

        if (n == 1) {
            cout << "2\n";
            continue; 
        }
        

        for (int i = 3; i < n+1; i++) {
            
            int id =0 ;
            for(;;) {
                if (arr[id] + arr[id+1] == i) {
                    tam += 2;
                    arr.insert(arr.begin()+ id+1, i);
                    id ++;
                }
                id++;
                if (id >= arr.size()) {
                    break;
                }
            }
        }
        cout << tam << "\n";
    }
}

